pages/event/[eventSlug]/schedules/index.vue

```
@@ -143,6 +144,9 @@ function isSelected(scheduleId: string) {
 function goBack() {
   navigateTo(`/event/${eventSlug}`);
 }
+
+const lastPreScreening = localStorage.getItem(`lastPreScreening_${eventSlug}`);
+const lastQuestionnairePreScreening = lastPreScreening ? JSON.parse(lastPreScreening) : undefined;
```

ESLint

[error] Replace `·?·JSON.parse(lastPreScreening)` with `⏎··?·JSON.parse(lastPreScreening)⏎·`

(prettier/prettier)